src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -282,10 +288,18 @@ export class ExtensionEditor extends BaseEditor {
 			})
 			.then(null, () => {
 				const p = append(this.content, $('p'));
-				p.textContent = localize('noReadme', "No README available.");
+				p.textContent = noContentCopy;
 			}));
 	}
 
+	private openReadme(extension: IExtension) {
+		return this.openMarkdown(extension, this.extensionReadme.get(), localize('noReadme', "No README available."));
+	}
+
+	private openChangelog(extension : IExtension) {
+		return this.openMarkdown(extension, this.extensionChangelog.get(), localize('noChangelog', "No CHANGELOG available."));
```

Comment:
This is all you have to do. We have tools that collect the new strings and send them off to the _localisers_.